airflow/providers/amazon/aws/auth_manager/views/auth.py

```diff
@@ -93,7 +93,7 @@ def login_callback(self):
             user_id=attributes["id"][0],
             groups=attributes["groups"],
             username=saml_auth.get_nameid(),
-            email=attributes["email"][0],
+            email=attributes["email"][0] if "email" in attributes else None,
```

Comment:
Is this a bugfix being included with the new tests?

Reply:
Correct

Reply:
Maybe also another PR? Wont push too hard though, I suppose it's fine if they're low impact.

Reply:
Yeah I supposed it made sense to include the fix as part of the same PR as the test? But I can also create a separate PR for it. I am fine either way